lib/index.js

```diff
@@ -27,4 +27,40 @@ function info() {
   return scriptsInfo;
 }
 
+function getScriptsInfo(pkgJSON) {
+  if ('scripts-info' in pkgJSON) {
+    return pkgJSON['scripts-info'];
+  }
+
+  var scriptsInfo = {};
+  Object.keys(pkgJSON.scripts || {})
+    .filter(scriptIsDescription)
+    .forEach(function(scriptName) {
+      scriptsInfo[scriptName.substr(1)] = getScriptDecription(pkgJSON.scripts[scriptName]);
+    });
+
+  return scriptsInfo;
+}
+
+function scriptIsDescription(scriptName) {
+  return scriptName[0] === '?';
+}
+
+function getScriptDecription(description) {
```

Comment:
Typo, should be "De**s**cription". IMO, this is better named as `getDescription`